src/services/symbolication.rs

```diff
@@ -2111,6 +2181,7 @@ mod tests {
             })],
             options: RequestOptions {
                 dif_candidates: true,
+                compare_stackwalking_methods: false,
```

Comment:
this isn't very important, but it might be more future-proof to do `RequestOptions { dif_candidates: true, ..Default::default() }` in these tests.